netlogo-gui/src/main/app/TabsMenu.scala

```diff
@@ -0,0 +1,41 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.app
+
+// TODO i18n lot of work needed here...
```

Comment:
I might be missing something, I don't see any I18N work to do in this file.  If this comment is kept, sign and date.  If it's from someone else in the past, I think it can just be removed.